tests/Microsoft.Fx.Portability.Tests/Analysis/AnalysisEngineTests.cs

```diff
@@ -25,7 +25,7 @@ public class AnalysisEngineTests
         [Fact]
         public void FindUnreferencedAssemblies_AllNulls()
         {
-            var engine = new AnalysisEngine(null, null);
+            var engine = new AnalysisEngine(null, null, null);
```

Comment:
Can you add some AnalysisEngineTests with a mock IPackageFinder?

Reply:
Added tests.